jmclient/jmclient/cryptoengine.py

```diff
@@ -1,10 +1,11 @@
 
 from collections import OrderedDict
+from bitcointx import base58
```

Comment:
This may seem a minor nit, but we want to always refer to bitcoin library stuff via the `jmbitcoin` package, not directly via the library it uses (indeed, we have recently switched from coincurve to bitcointx, and it could happen again), hence please use the kind of syntax seen here:

https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/18be93533372da3558f664312612dd462b3cc27c/scripts/obwatch/ob-watcher.py#L780

(i.e. refer to base58 as part of jmbitcoin even though it's pure passthrough to bitcointx)